.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+
+python:
+ - "3.7-dev"
```

Comment:
You don't really need quotes or tab here.

Reply:
Not a problem with 3.7-dev, nor with any of the current Python versions (will there be a Python 3.10?), but it's good practice because:

> In YAML, - 7.10 would evaluate to version 7.1, and not 7.10. ... 

> In YAML, 7.8 is a float, "7.8" is a string. Software version numbers should be strings, not floats. 

https://github.com/travis-ci/docs-travis-ci-com/pull/1537

But happy to remove them, let me know.